Add a basic json to yaml converter

lazka opened the pull request:
Nothing fancy, but has some nice defaults like preserving order and
indenting lists.

TingPing commented on flatpak-json2yaml.py:
```diff
@@ -0,0 +1,73 @@
+    with open(args.json_file, "rb") as h:
+        out_file = args.out_file
+        if out_file is None:
+            out_file = os.path.splitext(args.json_file)[0] + '.yaml'
```
We've been using the `yml` file extension mostly. Might as well be consistent.

lazka replied:
ok

TingPing commented:
A lot of flatpak json files use comments which are non-standard, it would be nice to not fail on those.

lazka commented:
> A lot of flatpak json files use comments which are non-standard, it would be nice to not fail on those.

ok, I'll have a look

lazka commented:
https://github.com/flathub/io.github.gillesdegottex.FMIT/blob/master/io.github.gillesdegottex.FMIT.json

```
./flatpak-json2yaml.py io.github.gillesdegottex.FMIT.json
Removed comment in line 14: '/* acs_alsa support */'
Removed comment in line 16: '/* acs_jack support */'
Removed comment in line 83: '/* acs_qt does not work on Linux */'
Removed comment in line 88: '/* must be done before calling qmake, or translations will not be installed (?) */'
```

Not sure if this is enough.. ?
